AUTHORS.txt

```diff
@@ -2,7 +2,6 @@ Lukáš Lalinský <lalinsky@gmail.com>
 Robert Kaye <rob@eorbit.net>
 Philipp Wolfer <ph.wolfer@googlemail.com>
 
-Picard icons by Carlin Mangar <openshift@gmail.com>
-
-The icons in this program are from the Tango Desktop Project
-<http://tango.freedesktop.org/>.
+Picard logo by <http://monkeydo.digital>, © 2015 MetaBrainz Foundation
+Some icons in this program are by Carlin Mangar <openshift@gmail.com>, the
```

Comment:
It is very vague, but IMO not more vague than what was already there. I looked over commits he has done, but he doesn't state on the icons he's added whether he's actually the creator of them or if they're taken from the Tango set (or if they're from some third place).